src/tallies/tally.cpp

```diff
@@ -419,17 +439,29 @@ void Tally::set_scores(const vector<std::string>& scores)
   bool energyout_present = energyout_filter_ != C_NONE;
   bool legendre_present = false;
   bool cell_present = false;
+  bool energy_present = false;
```

Comment:
Looks like this is not used -- can you remove it along with the check below?

Reply:
yes - good point